### PR TITLE
NOX-655, NOX-657 Generate Actions section and empty Add Form 

### PR DESCRIPTION
**Pull request by anna-naumova25**

### CODE REVIEW

**Comment by rochar** [comment]
@anna-naumova25  can you please add a screenshot of the end result for Crypto Cash?

File: src/Nox.Ui.Blazor.Lib/Components/Generic/Actions.razor [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿@using MudBlazor
+
+@namespace Nox.Ui.Blazor.Lib.Components.Generic
+
+@inherits ComponentBase
+
+<MudButton OnClick="OnAddClicked"
+           Variant="Variant.Filled"
+           StartIcon="@Icons.Material.Filled.Add"
+           Color="Color.Info">ADD</MudButton>
```
Comment (rochar) [inline src/Nox.Ui.Blazor.Lib/Components/Generic/Actions.razor]: should be localized?

**Review by rochar — CHANGES_REQUESTED** [review]
looks good

**Comment by DanielHodgeGH** [comment]
Hi @anna-naumova25 - are the code blocks on the razor pages there to make it easier for generator? I prefer to have the code on the related base cs file - but understand if makes it easier on generator side

**Comment by anna-naumova25** [comment]
Screenshot 2024-01-29 092417
Screenshot 2024-01-29 092438

File: src/Nox.Ui.Blazor.Lib/Components/Generic/Actions.razor (hunk 1 above)
Comment (anna-naumova25) [reply src/Nox.Ui.Blazor.Lib/Components/Generic/Actions.razor]: have localized all the existing buttons on ui

**Review by rochar — APPROVED** [review]

**Comment by anna-naumova25** [comment]
> Hi @anna-naumova25 - are the code blocks on the razor pages there to make it easier for generator? I prefer to have the code on the related base cs file - but understand if makes it easier on generator side

@DanielHodgeGH  I've moved codebase to a cs fil for AddForm. Absolutely agree on this, let's have the code in a separate file. Let's only keep it in .razor if the code block is really small.

**Review by DanielHodgeGH — APPROVED** [review]

**Pull request merged**